### DISCUSSION TITLE
我明明没设置账号密码，怎么要我输入账号密码啊？

### DISCUSSION THREAD

**Discussion by GEZiwww** [GaiZhenbiao/ChuanhuChatGPT#507] [Q&A] [answered]
"users": [
        ["username-0", "password-0"],
        ["username-0", "password-0"]
    ], 
    "advance_docs": {
        "pdf": {
            "two_column": false,
            "formula_ocr": true
        }
    }
}

**Comment by GEZiwww** [comment]
和川虎说的一样，输入api后导入config.json之后就变成了https://www.z4a.net/image/VHKe1N   这样。明明没开账号密码

**Comment by Keldos-Li** [comment, marked as answer]
抱歉，不设置用户名密码需要在config.json中删去
"users": [
["username-0", "password-0"],
["username-0", "password-0"]
],
这几行。

写在example中的是示例，但其实也变成了用户名和密码。……我们想想留空或者别的办法哈……不好意思带来困扰

**Comment by Keldos-Li** [comment]
现在可以留空了

**Reply by GEZiwww** [reply]
ok，问题也处理好了（，结果，它跟我讲api不对或者网络有问题，我提取了新的api也是一样，是不是gpt那边有什么东西啊？
